components/automate-ui/src/app/pages/user-details/user-details.component.ts

```diff
@@ -1,4 +1,4 @@
-import { combineLatest as combineLatest,  Observable ,  Subscription } from 'rxjs';
+import { combineLatest,  Observable ,  Subscription } from 'rxjs';
```

Comment:
Drive-by: remove redundant bit.

Reply:
Spaces look fishy, though 🐟  Since you've touched the line, let's do
```suggestion
import { combineLatest, Observable, Subscription } from 'rxjs';
```